flink-connectors/flink-sql-connector-hbase/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,23 @@
+flink-sql-connector-hbase
+Copyright 2014-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- org.apache.hbase:hbase-common:1.4.3
+- org.apache.hbase:hbase-protocol:1.4.3
+- org.apache.hbase:hbase-procedure:1.4.3
+- org.apache.hbase:hbase-client:1.4.3
+- org.apache.hbase:hbase-prefix-tree:1.4.3
+- org.apache.htrace:htrace-core:3.1.0-incubating
+- org.apache.zookeeper:zookeeper:3.4.14
+- commons-codec:commons-codec:1.10
+- commons-configuration:commons-configuration:1.7
+- commons-lang:commons-lang:2.6
+- commons-logging:commons-logging:1.1.3
+- com.google.guava:guava:12.0.1
+- com.google.protobuf:protobuf-java:2.5.0
```

Comment:
`protobuf-java` is not Apache Software License 2.0.

Reply:
You are right. It is BSD.